app/classes/auto_complete/for_user.rb

```diff
@@ -28,10 +28,10 @@ def matches_array(users)
       user[:name] = if user[:name].empty?
                       user[:login]
                     else
-                      "#{user[:login]} <#{user[:name]}>"
+                      User.unique_text_name(user)
                     end
```

Comment:
Why are you doing this rather than `user.unique_text_name`?

Reply:
Proposed changes here: https://github.com/MushroomObserver/mushroom-observer/pull/2665

Reply:
Thank you